app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  after_save :update_post_counter
```

Comment:
Great job on implementing the requested changes🤝

- [ ] As requested by the previous reviewer, I think you should use the `after_save` to call the `update_like_counter` in this `like` model. Instead of the `update_post_counter` method.